pstress/pstress-run.sh

```diff
@@ -1084,129 +1042,30 @@ EOF
       echoit "3 Node PXC Cluster failed to start after ${PXC_START_TIMEOUT} seconds. Will issue an extra cleanup to ensure nothing remains..."
       (ps -ef | grep 'n[0-9].cnf' | grep ${RUNDIR} | grep -v grep | awk '{print $2}' | xargs kill -9 >/dev/null 2>&1 || true)
       sleep 2; sync
-      SERVER_FAIL_TO_START_COUNT=$[ $SERVER_FAIL_TO_START_COUNT + 1 ]
```

Comment:
Looks like you have not created the PR on latest trunk. All the below changes were part of PSTRESS-96. You have reverted this PR. Please fix this